index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+    </style>
+    <title>portfolio</title>
+  </head>
+  <body>
+    <div class="flex-container">
+      <nav class="nav">
+        <h3 id="logo">My Logo</h3>
+        <img src="images/Union.png" alt="navbar" />
+      </nav>
+      <header>
+        <section class="hero">
+          <h2 id="intro">I'm Abdul Hadi Jafari Glad to see you!</h2>
```

Comment:
Please ensure that the main title (e.g., "I'm Marianne ...") is a `h1` tag and the subtitle uses a header tag `hx (examples: <h2>,<h3>, etc...)`